src/Tonic/Generator.h

```diff
@@ -76,6 +83,17 @@ namespace Tonic {
       obj->tick(frames, context);
     }
 
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(ControlGenerator pan);
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(float pan);
+
+  void ____recordAdditionAsParameter____(){ obj->hasBeenAdded = true; }
```

Comment:
Wow. Thanks for reading! You're good!

On Sat, Mar 7, 2015 at 1:34 PM, LB-- <notifications@github.com> wrote:

> In src/Tonic/Generator.h
> <https://github.com/TonicAudio/Tonic/pull/264#discussion_r25999820>:
>
> > @@ -76,6 +83,17 @@ namespace Tonic {
> >        obj->tick(frames, context);
> >      }
> >
> > +
> > +  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
> > +  // be replaced with a better panner that can handle mono or stereo.
> > +  Generator pan(ControlGenerator pan);
> > +
> > +  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
> > +  // be replaced with a better panner that can handle mono or stereo.
> > +  Generator pan(float pan);
> > +
> > +  void ____recordAdditionAsParameter____(){ obj->hasBeenAdded = true; }
>
> C++ reserves identifiers with two or more underscores in a row:
> http://stackoverflow.com/a/228797/1959975
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/TonicAudio/Tonic/pull/264/files#r25999820>.
>



-- 
===============
Morgan Packard
cell: (720) 891-0122
twitter: @morganpackard